packages/iTwinUI-react/package.json

```diff
@@ -108,8 +108,10 @@
   "lint-staged": {
     "*.{tsx,ts,jsx,js}": [
       "prettier --write",
-      "eslint --max-warnings=0 --fix",
       "node ../configs/copyrightLinter.js --fix"
+    ],
+    "*.{tsx,ts}": [
```

Comment:
Why separating eslint setting solves the issue?

Reply:
The issue happens because we are ignoring some js files in our eslint config. When we pass files that we have ignored back to eslint, it fails instead of just ignoring it. So excluding all js files is a way of avoiding the error by avoiding running eslint on those files.